.github/workflows/build_all.yml

```diff
@@ -54,29 +63,36 @@ jobs:
             source venv/bin/activate
           fi
 
-          python -m pip install pypi_json distro typer rich
+          python -m pip install pypi_json distro typer rich ./py-show
           python download_and_patch_sdist.py ${{ matrix.pkgdata[0]}}
+          ls -alh sdist/
+          ls -alh sdist/_amended
 
       - name: Install external dependencies
         run: |
           if [[ ${{ matrix.platform[1] }} == "conda-forge" ]]; then
             source ~/.bashrc && micromamba activate cfenv
+            case ${{ matrix.pkgdata[0] }} in
+              matplotlib)
+                export AR="${GCC_AR:gcc-ar}"
+              ;;
+              pyarrow)
+                micromamba install clangdev
+              ;;
```

Comment:
We'll probably need to define a dep URL for clang and friends. Or should this be part of `generic/llvm`?

Reply:
I think a `dep` for `clang` makes sense. It exists as a separate package in distros; `llvm` is a lot more heavyweight.